.github/actions/build-test-and-deploy/cloud-platform-deploy/action.yml

```diff
@@ -110,6 +114,19 @@ runs:
         brew install helm
         yq -i ".appVersion = \"${{ inputs.version }}\"" "${{ inputs.helm_dir }}/${{ inputs.helm_chart_name }}/Chart.yaml"
         helm dependency update "${{ inputs.helm_dir }}/${{ inputs.helm_chart_name }}"
+
+
+        # For services that have the startup/shutdown script enabled.
+        # Get pseudo-random, deterministic number for the cron start/shutdown minute.
+        # This is to help ensure the startup/shutdown pods do not all run at the exact same time - which can cause cluster auto-scalling issues.
+        if [[ "${{ inputs.scheduled_downtime_cron }}" == "true" ]]; then
+          cron_minute=$(echo -n "${{ inputs.k8s_deployment_name }}" | md5sum | awk '{print $1}' | tr -dc '0-9' | head -c 3)
+          cron_minute=$((cron_minute % 60))
+          startup_cron="$cron_minute 6 * * 1-5"
+          shutdown_cron="$cron_minute 21 * * 1-5"
+          helm_additional_args+=" --set generic-service.scheduledDowntime.startup=${startup_cron} --set generic-service.scheduledDowntime.shutdown=${shutdown_cron}"
```

Comment:
not sure how this works. You're creating a new bash variable called `helm_additional_args`, but lower down you're still referencing `inputs.helm_additional_args`.  I'd be surprised if github actions allow you to modify input arguments?

Reply:
I think it needs fixing.

Reply:
I've pushed a fix.